qt/scientific_interfaces/MultiDatasetFit/MDFDataController.cpp

```diff
@@ -5,12 +5,12 @@
 //   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 // SPDX - License - Identifier: GPL - 3.0 +
 #include "MDFDataController.h"
-#include "MDFAddWorkspaceDialog.h"
 #include "MultiDatasetFit.h"
 
 #include "MantidAPI/AnalysisDataService.h"
 #include "MantidAPI/MatrixWorkspace.h"
 #include "MantidAPI/WorkspaceGroup.h"
+#include "MantidQtWidgets/Common/AddWorkspaceDialog.h"
```

Comment:
did you expose this interface to Python?

Reply:
No because it is only ever opened from c++ at the moment